libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DatePicker.tsx

```diff
@@ -0,0 +1,146 @@
+// (C) 2025 GoodData Corporation
+
+import React, { forwardRef, useState } from "react";
+import { IntlShape } from "react-intl";
+import { WeekStart } from "@gooddata/sdk-model";
+import { Overlay } from "@gooddata/sdk-ui-kit";
+import {
+    DayPicker as DayPickerComponent,
+    DayPickerRangeProps,
+    SelectRangeEventHandler,
+    DayPickerProps,
+} from "react-day-picker";
+import enUS from "date-fns/locale/en-US/index.js";
+import de from "date-fns/locale/de/index.js";
+import es from "date-fns/locale/es/index.js";
+import fr from "date-fns/locale/fr/index.js";
+import ja from "date-fns/locale/ja/index.js";
+import nl from "date-fns/locale/nl/index.js";
+import pt from "date-fns/locale/pt/index.js";
+import ptBR from "date-fns/locale/pt-BR/index.js";
+import zhCN from "date-fns/locale/zh-CN/index.js";
+import ru from "date-fns/locale/ru/index.js";
+import it from "date-fns/locale/it/index.js";
+import enGB from "date-fns/locale/en-GB/index.js";
+import frCA from "date-fns/locale/fr-CA/index.js";
+import enAU from "date-fns/locale/en-AU/index.js";
+import fi from "date-fns/locale/fi/index.js";
+
+import { mergeDayPickerProps } from "./utils.js";
+import { IDateRange } from "./types.js";
+
+const convertedLocales: Record<string, Locale> = {
+    "en-US": enUS,
+    "de-DE": de,
+    "es-ES": es,
+    "fr-FR": fr,
+    "ja-JP": ja,
+    "nl-NL": nl,
+    "pt-BR": ptBR,
+    "pt-PT": pt,
+    "zh-Hans": zhCN,
+    "ru-RU": ru,
+    "it-IT": it,
+    "es-419": es,
+    "en-GB": enGB,
+    "fr-CA": frCA,
+    "zh-Hant": zhCN,
+    "en-AU": enAU,
+    "fi-FI": fi,
+    "zh-HK": zhCN,
+};
+
+const ALIGN_POINTS = [{ align: "bl tl", offset: { x: 0, y: 1 } }];
+
+const convertLocale = (locale: string): Locale => convertedLocales[locale];
+
+function convertWeekStart(weekStart: WeekStart): DayPickerProps["weekStartsOn"] {
+    switch (weekStart) {
+        case "Sunday":
+            return 0;
+        case "Monday":
+            return 1;
+        default:
+            throw new Error(`Unknown week start ${weekStart}`);
+    }
+}
+
+export const DayPicker = forwardRef<
+    HTMLDivElement,
+    {
+        mode: "from" | "to";
```

Comment:
put it to interface

Reply:
Fixed. I already had interface that I could use.